controllers/storagecluster/reconcile.go

```diff
@@ -87,11 +87,39 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 
 var storageClusterFinalizer = "storagecluster.ocs.openshift.io"
 
-var validTopologyLabelKeys = []string{
-	"failure-domain.beta.kubernetes.io",
-	"failure-domain.kubernetes.io",
-	"kubernetes.io/hostname",
-	"topology.rook.io",
+const labelZoneRegionWithoutBeta = "failure-domain.kubernetes.io/region"
+const labelZoneFailureDomainWithoutBeta = "failure-domain.kubernetes.io/zone"
+const labelRookPrefix = "topology.rook.io"
+
+func getValidTopologyLabelKeys() []string {
```

Comment:
Why create a getter to generate a string list every time instead of retaining the global variable and using the getter to return the global variable? Having the global variable with one item per line (as before) also helps keep diffs clean.

Reply:
Done